Write a python script to generate a UUID for the fxa email within the integration-tests.

It seems that the UUID package will randomly not resolve. This add extra flakiness to the integration tests on circleci that really shouldn't exist. We should build a simple python script to generate an fxa email the tests can use.

Log of the failure:
```
Traceback (most recent call last):
  File "manage.py", line 11, in <module>
    execute_from_command_line(sys.argv)
  File "/usr/local/lib/python3.6/site-packages/django/core/management/__init__.py", line 381, in execute_from_command_line
    utility.execute()
  File "/usr/local/lib/python3.6/site-packages/django/core/management/__init__.py", line 375, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/usr/local/lib/python3.6/site-packages/django/core/management/base.py", line 323, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/usr/local/lib/python3.6/site-packages/django/core/management/base.py", line 364, in execute
    output = self.handle(*args, **options)
  File "/code/src/olympia/landfill/management/commands/generate_default_addons_for_frontend.py", line 80, in handle
    serializer = GenerateAddonsSerializer()
  File "/code/src/olympia/landfill/serializers.py", line 52, in __init__
    self.fxa_id = self._create_fxa_user()
  File "/code/src/olympia/landfill/serializers.py", line 63, in _create_fxa_user
    fxa_account.password)
  File "/usr/local/lib/python3.6/site-packages/fxa/core.py", line 64, in create_account
    resp = self.apiclient.post(url, body)
  File "/usr/local/lib/python3.6/site-packages/fxa/_utils.py", line 335, in post
    return self.request("POST", url, json, **kwds)
  File "/usr/local/lib/python3.6/site-packages/fxa/_utils.py", line 321, in request
    raise fxa.errors.ClientError(body)
fxa.errors.ClientError: Account already exists
Makefile-docker:161: recipe for target 'setup-ui-tests' failed
```

--- a/.circleci/config.yml
+++ b/.circleci/config.yml
@@ -106,9 +106,7 @@ jobs:
           command: |
             set -x
             sudo sysctl -w vm.max_map_count=262144
-            # Install UUID
-            sudo apt-get update -qqy && sudo apt-get -qqy install uuid
-            export UITEST_FXA_EMAIL=uitest-$(uuid)@restmail.net
+            export UITEST_FXA_EMAIL=$(python ./tests/ui/scripts/generate_fxa_email.py)
             # Pull images
             docker-compose -f docker-compose.yml -f tests/ui/docker-compose.selenium.yml pull --quiet
             # Start Containers

--- a/.circleci/config.yml
+++ b/.circleci/config.yml
@@ -113,7 +113,7 @@ jobs:
           command: |
             set -x
             sudo sysctl -w vm.max_map_count=262144
-            export UITEST_FXA_EMAIL=$(python ./tests/ui/scripts/generate_fxa_email.py)
+            export UITEST_FXA_EMAIL=$(python3 ./tests/ui/scripts/generate_fxa_email.py)
             # Pull images
             docker-compose -f docker-compose.yml -f tests/ui/docker-compose.selenium.yml pull --quiet
             # Start Containers

--- a/.circleci/config.yml
+++ b/.circleci/config.yml
@@ -85,12 +85,8 @@ jobs:
             - uitest-cache-{{ checksum "requirements/uitests.txt" }}
             - uitest-cache-
       - run:
-          name: Install python 3
-          command: |
-            sudo apt install software-properties-common
-            sudo add-apt-repository ppa:deadsnakes/ppa
-            sudo apt update
-            sudo apt install python3.7
+          name: "Set Python Version"
+          command: pyenv global 3.7.0
       - run:
           name: Install Docker Compose
           command: |
@@ -113,7 +109,7 @@ jobs:
           command: |
             set -x
             sudo sysctl -w vm.max_map_count=262144
-            export UITEST_FXA_EMAIL=$(python3 ./tests/ui/scripts/generate_fxa_email.py)
+            export UITEST_FXA_EMAIL=$(python ./tests/ui/scripts/generate_fxa_email.py)
             # Pull images
             docker-compose -f docker-compose.yml -f tests/ui/docker-compose.selenium.yml pull --quiet
             # Start Containers

--- a/.circleci/config.yml
+++ b/.circleci/config.yml
@@ -85,6 +85,13 @@ jobs:
             - uitest-cache-{{ checksum "requirements/uitests.txt" }}
             - uitest-cache-
       - run:
+          name: Install python 3
+          command: |
+            sudo apt install software-properties-common
+            sudo add-apt-repository ppa:deadsnakes/ppa
+            sudo apt update
+            sudo apt install python3.7
+      - run:
           name: Install Docker Compose
           command: |
             # using curl to install for better ci -> local compatibility

--- a/Dockerfile
+++ b/Dockerfile
@@ -42,8 +42,6 @@ RUN apt-get update && apt-get -t stretch install -y \
         librsvg2-bin \
         # Use pngcrush to optimize the PNGs uploaded by developers
         pngcrush \
-        # our makefile and ui-tests require uuid to be installed
-        uuid \
         # Use libmaxmind for speedy geoip lookups
         libmaxminddb0                    \
         libmaxminddb-dev                 \

--- a/Dockerfile.deploy
+++ b/Dockerfile.deploy
@@ -35,7 +35,6 @@ RUN apt-get update && apt-get -t stretch install -y \
         libmagic-dev                     \
         libssl-dev                       \
         nodejs                           \
-        uuid-dev                         \
         # Git, because we're using git-checkout dependencies
         git                              \
         # Dependencies for mysql-python

--- a/tests/ui/scripts/generate_fxa_email.py
+++ b/tests/ui/scripts/generate_fxa_email.py
@@ -2,9 +2,7 @@ import uuid
 
 
 def create_fxa_email():
-    id = uuid.uuid1()
-    email = f"uitest-{id}@restmail.net"
-    print(f"{email}")
+    print(f"uitest-{uuid.uuid1()}@restmail.net")
 
 
 if __name__ == "__main__":

--- a/tests/ui/scripts/generate_fxa_email.py
+++ b/tests/ui/scripts/generate_fxa_email.py
@@ -1,0 +1,10 @@
+import uuid
+
+
+def create_fxa_email():
+    id = uuid.uuid1()
+    email = f"uitest-{id}@restmail.net"
+    print(f"{email}")
+
+if __name__ == "__main__":
+    create_fxa_email()

--- a/tests/ui/scripts/generate_fxa_email.py
+++ b/tests/ui/scripts/generate_fxa_email.py
@@ -6,5 +6,6 @@ def create_fxa_email():
     email = f"uitest-{id}@restmail.net"
     print(f"{email}")
 
+
 if __name__ == "__main__":
     create_fxa_email()

